### PR TITLE
Milestone003:Contactform

### PR DESCRIPTION
**Pull request by borrnelsonk**
The following functionalities were added to the contact form;

1. The markup to collect user's Name, email and message. ✔️ 
2. Form validation of user input. ✔️ 
3. Directing user submissions to formspree . ✔️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,6 +72,30 @@ <h1 class="worktext">My Recent Works</h1>
...
+                    <!-- <label for="message">Enter text here</label> -->
+                    <textarea id="message" name="usermessage" required maxlength="500" placeholder="Enter text here"></textarea>
+                </li>
+                <li class="buttonlabel">
+                    <button class="fbutton" type="submit"><ins class="btext">Get in touch</ins></button>
+                </li>
+                </ul>
+            </form>
+            
+        </footer>
```
Comment (cynthiainga) [inline index.html:98]: - [ ] Please kindly make sure your design stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), also kindly fix the empty space to the right of your website 🙃 🙃 
- [ ] Please kindly add the socials media icons to the end of your website 😉

Your design
Screen Shot 2022-08-30 at 18 08 56

Figma design
Screen Shot 2022-08-30 at 18 08 09

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @borrnelsonk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by borrnelsonk** [comment]
Thankyou @cynthiainga for your review.  💘 I have incorporated the changes you suggested.
Please check out. 👍

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @borrnelsonk , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost3

 ## Highlights
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

*** Kindly, I see that you made an effort to improve your site, but I still see some details to improve so that your site is pro! 😉

- A horizontal scroll is created that you should remove, find what element is producing it and adjust it.

- The "See project" and "get in touch" buttons do not have the correct vertical and horizontal padding, please adjust them.

- Do not use static widths (for example in the headline class), I recommend using percentages of 100% so that your website adapts to any device size.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by borrnelsonk** [comment]
Thankyou so much @mgmediaweb for your review. Let me make the changes and revert.

**Comment by borrnelsonk** [comment]
Thankyou so much @mgmediaweb for your review. I have made the changes based on the suggestions you made and it has really improved my website. 😄 
However based on the scope of this project, I have not made all sections of the site to adapt to different screens ❌ as I will be working on it on my next project.
I hope this milestone is worthy of your approval.  ☑️

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @borrnelsonk ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

- [ ] Kindly remove and fix both the vertical and horizontal scrolls to improve your design
- [ ] You may also remove the empty space at the bottom of your design. Kindly fix the `margin bottom`.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by borrnelsonk** [comment]
Thankyou so much @1GORDON

**Review by sajid-munawar — COMMENTED** [review]
### Hi @borrnelsonk , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**